its/src/test/helmSuite/helm.test.ts

```diff
@@ -30,9 +29,8 @@ suite('Helm Test Suite', () => {
 
     const diags = await waitForSonarLintDiagnostics(fileUri);
 
-    assert.strictEqual(diags.length, 2);
-    assert.strictEqual(diags[0].message, "Specify a storage limit for this container.");
-    assert.strictEqual(diags[1].message, "Specify a storage request for this container.");
+    assert.strictEqual(diags.length, 1);
+    assert.strictEqual(diags[0].message, "Specify a storage request for this container.");
```

Comment:
Is this expected? I think it would be great to have a confirmation from the Cloud Native squad 😅